pkg/asset/k8s.go

```diff
@@ -105,40 +109,67 @@ func newDynamicAssets(conf Config) Assets {
 	return assets
 }
 
+const validBootstrapTokenChars = "0123456789abcdefghijklmnopqrstuvwxyz"
+
+// newBootstrapToken constructs a bootstrap token in conformance with the following format:
+// https://kubernetes.io/docs/admin/bootstrap-tokens/#token-format
+func newBootstrapToken() (id string, secret string, err error) {
+	token := make([]byte, 6+16)
```

Comment:
nit: not immediately clear what 6+16 is (id+secret).

Reply:
Added comment.